Makefile

```diff
@@ -26,7 +26,9 @@ GOHOSTOS ?= $(shell go env GOHOSTOS)
 # Allow limiting the scope of the e2e tests. By default run everything
 FOCUS ?= .*
 # number of parallel test
-E2E_NODES ?= 3
+E2E_NODES ?= 4
```

Comment:
`?=` is evil, it allows for empty strings. `:=` is safer.

Reply:
most of the variables could be replaced by env variables, using `:=` avoids that. Also, the scripts we call check for the required variables

Reply:
`:=` should handle env vars as well. What version of make do you have installed?

Reply:
$ make -v
GNU Make 4.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2014 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.